### ISSUE TITLE
Getting error TypeError: fontFamily is undefined in Firefox theme.js:7993:11

### ISSUE THREAD

**Issue by gieronkrzysztof** [tinymce/tinymce#4368]
**Do you want to request a *feature* or report a *bug*?**
Report a bug

**What is the current behavior?**
I'm getting error TypeError: fontFamily is undefined in theme.js:7993:11 file. It happens only in Firefox, no error in Chrome.
![image](https://user-images.githubusercontent.com/27882523/39431175-32670504-4c90-11e8-9ffa-1f21e81ff4e4.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**
It happens on init of tinymce component. My init config is following:

```
tinymce.init({
            selector: 'textarea.mce.line-item-template',
            plugins: 'advlist autolink anchor code textcolor colorpicker contextmenu directionality hr image importcss template',
            toolbar: 'anchor code forecolor backcolor ltr rtl image template',            
            menubar: 'insert tools view format',
            contextmenu: 'link image inserttable | cell row column deletetable',
            contextmenu_never_use_native: false,
            forced_root_block: false,
            protect: [
                /\<\/?(td|tr)\>/g, 
                /\<\/?(td|tr) class=\"[\w\s-]+\"\>/g,
            ],
            verify_html: false
 });
```

**What is the expected behavior?**
No errors in firefox.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Firefox Quantum 59.0.2 (64-bit) running on Windows, TinyMCE Version 4.7.11

**Comment by crowake13** [comment]
I get the same issue when I initialise tinymce on a textarea located in a tab, 
if the textarea is outside of the tab I get no error.  

For the tabs I'm using [angular-ui-bootstrap](https://github.com/angular-ui/bootstrap/blob/gh-pages/ui-bootstrap-tpls-0.11.2.js).

**Comment by spocke** [comment]
Right I managed to reproduce this now will look into th this to get it solved.

**Comment by crowake13** [comment]
@spocke , it is great that you were able to reproduce the problem. 

Please let us know an approximate waiting time for a fix and/or is there maybe a workaround... It is very important for us to know how fast we can fix this. Even if it is not gonna be done fast, we need to know so we can plan future actions accordingly. 

Thank you in advanced.

**Comment by trevorjay** [comment]
This seems to have been fixed in 5.0.8 but is still an issue for the 4x stream (problem occurs on 4.9.10).

**Comment by trevorjay** [comment]
Further testing shows the latest releases work with their recommended versions of Angular. That is: 5.0.8 and above is working with Angular 5 and 4.9.10 is working with Angular 4 (via @tinymce/tinymce-angular-legacy). I am closing this issue. If anyone can still reproduce this, please let us know.
